management_api_app/api/routes/workspaces.py

```diff
@@ -110,6 +110,21 @@ async def retrieve_users_active_workspace_services(user=Depends(get_current_user
     return WorkspaceServicesInList(workspaceServices=workspace_services)
 
 
+@router.patch("/workspaces/{workspace_id}/workspace-services/{service_id}", response_model=WorkspaceServiceInResponse, name=strings.API_UPDATE_WORKSPACE_SERVICE)
+async def patch_workspace_service(workspace_service_patch: WorkspaceServicePatchEnabled,
+                                  workspace_service_repo=Depends(get_repository(WorkspaceServiceRepository)),
+                                  user=Depends(get_current_user),
+                                  workspace_service=Depends(get_workspace_service_by_id_from_path),
+                                  workspace=Depends(get_deployed_workspace_by_workspace_id_from_path)) -> WorkspaceInResponse:
+
+    role = get_user_role_in_workspace(user, workspace)
+    if role != WorkspaceRole.Owner:
```

Comment:
@marrobi who should be able to disable a workspace service? Shouldn't an admin be able to do this even if they are not workspace owners?

Reply:
For now Workspace Owner only. As that's the persona that can create and delete workspace services. 

We may need the ability to configure what operations each role can do on a per deployment basis at a later point in time.